qcodes/tests/validators/test_multi_type.py

```diff
@@ -21,17 +32,60 @@ def test_good():
 
     assert not MultiType(Strings(), Enum(1, 2)).is_numeric
 
```

Comment:
I think we might as well split this into two tests e.g. test_good_or and test_good_and

Reply:
Fixed.